drivers/SmartThings/zigbee-sensor/config.yml

```diff
@@ -0,0 +1,6 @@
+name: 'Zigbee Sensor'
```

Comment:
@varzac @greens @tpmanley Is the name of this driver(zigbee-Sensor) okay? Should we add the IAS explicitly?
like 'zigbee-ias-sensor' and the device label also.(now, it is 'IAS Zigbee Generic Sensor')
I wonder your opinion.

Reply:
I think it's OK as is